integration_tests/ctesque/src/test/java/android/graphics/BitmapTest.java

```diff
@@ -67,7 +67,7 @@ public void testExtractAlpha() {
     Bitmap ret = bitmap.extractAlpha();
     int source = bitmap.getPixel(10, 20);
     int result = ret.getPixel(10, 20);
-    assertThat(Color.alpha(result)).isEqualTo(Color.alpha(source));
+    assertThat(result).isEqualTo(Color.alpha(source));
```

Comment:
nit: this makes the test fail on a real device with the error:

```
expected: 255
but was : -16777216
	at android.graphics.BitmapTest.testExtractAlpha(BitmapTest.java:70)
```

I think there is a way to run BitmapTest on an emulator (instrumentation test) using gradle, but I'm not 100% sure how to do it.

Reply:
Yes, we can use instrumentation to run it, and use emulator for CI. The Google has released the headless emulator for CI very long ago.

Reply:
Also the [ShadowBitmap#extractAlpha](https://github.com/robolectric/robolectric/blob/7d898a88305d06edc4d9d6752bc96c5786ed8a63/shadows/framework/src/main/java/org/robolectric/shadows/ShadowBitmap.java#L496) just uses `Color.alpha` to get alpha value, and assigns it to new `Bitmap` as pixel value. So the modification is meaningful, I think, on current occasion.

Reply:
I will test real `Bitmap.extractAlpha` behavior on official emulator to check it just extract alpha value as pixel value or keeps alpha channel only, and set other channels to zero as pixel value. It will determine we should modify `BitmapTest.testExtractAlpha` or `ShadowBitmap.extractAlpha`, I think.